people/people.go

```diff
@@ -0,0 +1,35 @@
+package people
+
+import (
+	"fmt"
+	"sort"
+)
+
+type People []Person
+
+func (pp People) Append(ps Person) People {
+	return append(pp, ps)
+}
+
+func (pp People) String() (r string) {
+	for k, v := range pp {
+		r += fmt.Sprintf("%d. %s %s (%s)\n", k+1, v.firstName, v.lastName, v.birthDay)
+	}
+
+	return
+}
+
+func (pp People) Sort() {
```

Comment:
The main reason for this task, write structure which implicitly implements some interface due to this please implement interface https://golang.org/pkg/sort/#Interface and use sort.Sort function

Reply:
I think `sort.Slice` looks much better in this case because we use the default params of `Len()` and `Swap()` methods and have no reasons to rewrite them, but you're right in terms of home-task and I really didn't implement an interface, lol.
So fixed https://github.com/VSKut/go-temp/pull/7/commits/bc2a688c6f839b64f7dc620d65eae18eacf0fd14
Thanks.

Reply:
It's more about different things, to sort.Slice you have to pass less function always,  implementation of sort.Interface allow you incapsulate this and just pass slice to function.